### DISCUSSION TITLE
Privacy Feature - zoom

### DISCUSSION THREAD

**Discussion by bobblkabb** [hyprwm/Hyprland#646] [General]
Hello,

I would be very interested in a zoom feature or some kind of magnification app.
I read my daily newspaper online. The articles are relatively small and can't be read properly until you click on them and the article opens. For the sake of anonymity, I forgo this and instead zoom the articles in the page overview.
Wayfire is the only software where I can do this with Wayland.

**Comment by vaxerski** [comment]
It's planned, but it's a very niche feature that would require quite a big amount of code and reworking to make possible, so it's not the priority now.

**Reply by bobblkabb** [reply]
Glad to hear. Looking forward.

**Reply by edrex** [reply]
What about just changing the scale factor for the window?

**Reply by edrex** [reply]
I just found this. See #1282 (should have been discussion, oops)

**Comment by aquifolly** [comment]
it could be in a separate repo, like how hyprpicker/hyprpaper are

**Reply by aquifolly** [reply]
could be called hyprzoom

**Reply by vaxerski** [reply]
I think what they meant was something like the magnifying glass tool on windows, which cannot be made with a separate client on wayland.

**Comment by LexNastin** [comment]
Just waking up this thread to ask if there has been any progress on this? It would be very nice to have! I understand the complexities tho, so no rush!

**Reply by deed0s** [reply]
I haven't actually tried it out yet but [this commit](https://github.com/hyprwm/Hyprland/commit/b05ff89c76b2c9454c76daf2120f4ccb076a018d) added a _**cursor_zoom_factor**_ variable that acts like a magnifying glass around the cursor, there's also _**cursor_zoom_rigid**_ which dictates whether the zoom should follow the cursor rigidly or loosely, the wiki entry can be found [here](https://wiki.hyprland.org/Configuring/Variables/#:~:text=1-,cursor_zoom_factor,-the%20factor%20to)

**Reply by LexNastin** [reply]
Just tested. This is exactly it! Thanks for pointing out :D
